projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1943,17 +1943,19 @@ public function get_api_data( $blog_id ) {
 	/**
 	 * Get the Publicize shares info.
 	 *
+	 * This function is overwritten in class-publicize-wpcom.php
+	 *
 	 * @param string $blog_id The WPCOM blog_id for the current blog.
 	 * @return ?array
 	 */
-	public function get_publicize_shares_info( $blog_id ) {
```

Comment:
This function is overriden in wpcom fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Snqzva%2Qcyhtvaf%2Schoyvpvmr%2Schoyvpvmr%2Qjcpbz.cuc%3Se%3Qp72qssq8%232572-og